tests/e2e/templates/user.yaml.tmpl

```diff
@@ -0,0 +1,11 @@
+apiVersion: kafka.banzaicloud.io/v1alpha1
+kind: KafkaUser
+metadata:
+  name: {{or .Name "test-user"}}
+  namespace: {{or .Namespace "kafka" }}
+spec:
+  clusterRef:
+    name: kafka
```

Comment:
**Question**: Shouldn't this be parametrized?

Reply:
oh yeah, I1ll do that real quick

Reply:
Thanks